tests/Fixer/ArrayNotation/YieldFromArrayToYieldsFixerTest.php

```diff
@@ -0,0 +1,196 @@
+            '<?php function f() {
+                yield from [
+                    [1, 2],
+                    [3, 4],
+                ];
+            }',
+        ];
+
+        yield [
+            '<?php function f() {
+                 '.'
+                    yield array(1, 2);
+                    yield array(3, 4);
+                '.'
+            }',
+            '<?php function f() {
+                yield from [
+                    array(1, 2),
+                    array(3, 4),
+                ];
+            }',
+        ];
+
+        yield [
+            '<?php function f() {
+                 '.'
+                    yield 1;
+                    yield 2;
+                    yield 3;
+                '.'
+            }',
+            '<?php function f() {
+                yield from [
+                    1,
+                    2,
+                    3,
+                ];
+            }',
+        ];
+
+        yield [
+            '<?php function f() {
+                 '.'
+                    yield random_key() => true;
+                    yield "foo" => foo(1, 2);
+                    yield "bar" => function ($x, $y) { return max($x, $y); };
+                    yield "baz" => function () { yield [1, 2]; };
+                '.'
+            }',
+            '<?php function f() {
+                yield from [
+                    random_key() => true,
+                    "foo" => foo(1, 2),
+                    "bar" => function ($x, $y) { return max($x, $y); },
+                    "baz" => function () { yield [1, 2]; },
+                ];
+            }',
+        ];
+
+        yield [
+            '<?php
+                function f1() {  yield 0; yield 1; yield 2; }
+                function f2() {  yield 3; yield 4; yield 5; }
+                function f3() {  yield 6; yield 7; yield 8; }
+            ',
+            '<?php
+                function f1() { yield from [0, 1, 2]; }
+                function f2() { yield from [3, 4, 5]; }
+                function f3() { yield from [6, 7, 8]; }
+            ',
+        ];
+
+        yield [
+            '<?php
+                function f1() {  yield 0; yield 1; }
+                function f2() {  yield 2; yield 3; }
+                function f3() {  yield 4; yield 5; }
+                function f4() {  yield 6; yield 7; }
+                function f5() {  yield 8; yield 9; }
+            ',
+            '<?php
+                function f1() { yield from array(0, 1); }
+                function f2() { yield from [2, 3]; }
+                function f3() { yield from array(4, 5); }
+                function f4() { yield from [6, 7]; }
+                function f5() { yield from array(8, 9); }
+            ',
+        ];
+
+        yield [
+            '<?php
+                function foo() {
+                     '.'
+                        yield "this element is regular string";
+                        yield yield from ["here", "are", "nested", "strings"];
```

Comment:
I know I was coming with the code variant in `before fixing` version here, but is `yield yield from ...` is valid and meaningful syntax?
would it be same as `yield from` ? should we unpack?

Reply:
Using "return value" of `yield from` does not make any sense in the first place.